### PR TITLE
Bug 1910805: add translation for pipeline status

### PR DESCRIPTION
**Pull request by vikram-raj**
**Fixes**: 
https://issues.redhat.com/browse/ODC-5292

**Analysis / Root cause**: 
Translation for Pipeline status and breadcrumb text is missing.

**Solution Description**: 
Add translation for Pipeline status and breadcrumb text.

### CODE REVIEW

**Comment by debsmita1** [comment]
The dropdown items are not translated
items

File: frontend/packages/console-shared/src/hooks/useTabbedTableBreadcrumb.ts [hunk 1]
```diff
@@ -32,6 +34,6 @@ export const useTabbedTableBreadcrumbsFor = (
       name: kindObj.labelPlural,
```
[the remark this answers is not in the document]
Comment (debsmita1) [reply frontend/packages/console-shared/src/hooks/useTabbedTableBreadcrumb.ts:34]: oh right! my bad

**Comment by debsmita1** [comment]
t3
t1
t

these are missed

**Comment by vikram-raj** [comment]
Added the translation for pipelines create options.

**Review by jerolimov — CHANGES_REQUESTED** [review]
@vikram-raj Also with this PR the pipeline status fields are not translated. On the Pipeline list, PipelineRun list as well on the TaskRun list:

image

In the breadcrumb the word "Pipelines" is not translated on the overview page and on the pipeline detail page. But it is translated in the navigation. We do this also for Build Configs and Helm Release, so I think we should do this also for Pipelines:

image

image

**Comment by vikram-raj** [comment]
@jerolimov Added the translation for the `Pipeline` page title. And for the status and breadcrumbs, we are using the component from the public and console-shared. So, adding the translation for status and breadcrumb requires works in other packages as well.

**Comment by andrewballantyne** [comment]
@vikram-raj Isn't it explicitly in your ticket to support the status values? Am I missing something or misunderstanding something?

Screen Shot 2021-01-12 at 11 26 17 AM
Screen Shot 2021-01-12 at 11 26 28 AM

**Comment by vikram-raj** [comment]
@jerolimov @andrewballantyne Added translation for status as well.

File: frontend/packages/pipelines-plugin/src/components/pipelines/modals/triggers/TriggerBindingSection.tsx [hunk 2]
```diff
@@ -62,10 +62,8 @@ const TriggerBindingSection: React.FC = () => {
               isHidden={false}
               className="odc-trigger-binding-section__variable-help-text"
             >
-              <Trans t={t} ns="pipelines-plugin">
-                Use this format when referencing variables in this form:{' '}
-                <code>{`$(${paramPrefix}parameter)`}</code>
-              </Trans>
+              {t('pipelines-plugin~Use this format when referencing variables in this form: ')}
+              <code>{`$(${paramPrefix}parameter)`}</code>
```
Comment (jerolimov) [inline frontend/packages/pipelines-plugin/src/components/pipelines/modals/triggers/TriggerBindingSection.tsx:66]: :+1:

**Review by jerolimov — APPROVED** [review]
Tested this locally, pipeline status fields are now translated.

/lgtm

**Comment by jerolimov** [comment]
/assign @andrewballantyne 

Andrew, FYI, we discussed that we do not translate the status in the header field at the moment because this needs a unnecessary big refactoring. This field is also not translated for Pods etc. So there are still many small things to-do for 4.7.z or 4.8. Are you fine with this?

image

**Comment by andrewballantyne** [comment]
> /assign @andrewballantyne
> 
> Andrew, FYI, we discussed that we do not translate the status in the header field at the moment because this needs a unnecessary big refactoring. This field is also not translated for Pods etc. So there are still many small things to-do for 4.7.z or 4.8. Are you fine with this?
> 
> image

That's not ideal but can be done separately of this PR. Please open up a forum to discuss with the Admin team in #tmp-i18n slack room. See if they are aware of / already addressing the status in the header.

**Review by andrewballantyne — APPROVED** [review]

**Pull request merged**